bsip-0087.md

```diff
@@ -0,0 +1,36 @@
+    BSIP: 
+    Title: Force Settlement Fee Ratio
+    Authors:
+      Jerry Liu bitcrab@qq.com
+    Status: Draft
+    Type: Protocol
+    Created: 2020-02-23
+    Discussion: https://github.com/bitshares/bsips/issues/260
+    Worker: TBD
+
+# Abstract
+This BSIP provide a solution to charge fee from force settlement.
+
+# Motivation
+Force settlement is an important part in smartcoin design, it provides the power to smartcoin owner to ask for collaterals with reference to feed price, and then guarantees the peg of smartcoin.
```

Comment:
"smartcoin owner" is used several times in the document, but meaning different group of people. I think it should be "smartcoin holders" here.